drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/test/java/org/drools/workbench/screens/scenariosimulation/client/domelements/ScenarioHeaderTextAreaDOMElementTest.java

```diff
@@ -72,6 +74,20 @@ public void flushMetadataNotEqualsValue() {
         verify(scenarioHeaderTextAreaDOMElement, times(1)).internalFlush(eq(MULTIPART_VALUE));
     }
 
+    @Test
+    public void flushNullString() {
+        scenarioHeaderTextAreaDOMElement.flush(null);
+        verify(eventBusMock, times(1)).fireEvent(isA(ScenarioNotificationEvent.class));
+        verify(scenarioHeaderTextAreaDOMElement, never()).internalFlush(any());
+    }
+
+    @Test
+    public void flushEmptyString() {
+        scenarioHeaderTextAreaDOMElement.flush("");
+        verify(eventBusMock, times(1)).fireEvent(isA(ScenarioNotificationEvent.class));
+        verify(scenarioHeaderTextAreaDOMElement, never()).internalFlush(any());
+    }
+
```

Comment:
@yesamer 
Could you please add a test with a string containing only spaces ?

Reply:
@gitgabrio Done!